packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```
@@ -1,47 +1,49 @@
+# Generated from foreman-tasks-0.15.1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman-tasks
+%global plugin_name foreman-tasks
+%global foreman_min_version 1.17.0
 
-%global foreman_bundlerd_dir /usr/share/foreman/bundler.d
-
-Summary: Tasks support for Foreman with Dynflow integration
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.15.0
-Release: 2%{?foremandist}%{?dist}
-Group: Development/Libraries
-License: GPLv3
+Version: 0.15.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman plugin for showing tasks information for resoruces and users
+Group: Applications/Systems
+License: GPL-3.0-or-later
 URL: https://github.com/theforeman/foreman-tasks
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Source1: %{gem_name}.logrotate
-Requires: foreman >= 1.17.0
 
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core)
-Requires: %{?scl_prefix}rubygem(dynflow) >= 1.2.1
-Requires: %{?scl_prefix}rubygem(dynflow) < 2.0
-Requires: %{?scl_prefix}rubygem(get_process_mem)
-Requires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
-Requires: %{?scl_prefix}rubygem(parse-cron) < 0.2.0
+Requires: %{?scl_prefix}rubygem(dynflow) >= 1.2.2
 Requires: %{?scl_prefix_ror}rubygem(sinatra)
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}rubygems
-
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
+Requires: %{?scl_prefix}rubygem(parse-cron) < 0.2
+Requires: %{?scl_prefix}rubygem(get_process_mem)
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks-core)
-BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.1
-BuildRequires: %{?scl_prefix}rubygem(dynflow) < 2.0
-BuildRequires: %{?scl_prefix}rubygem(get_process_mem)
+BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.2
+BuildRequires: %{?scl_prefix_ruby}rubygem(sinatra)
```

This is a mistake in our SCL prefix file: https://github.com/theforeman/foreman-packaging/pull/3631

```suggestion
BuildRequires: %{?scl_prefix_ror}rubygem(sinatra)
```